qualification/report/generate_pdf.py

```diff
@@ -876,17 +894,21 @@ def main():
     parser.add_argument("input", help="Output of pytest --report-log=...")
     parser.add_argument("pdf", help="PDF file to write")
     parser.add_argument("-c", "--commit-id", action="store_true", help="Output commit ID of katgpucbf image")
+    parser.add_argument("--generate-procedure-doc", action="store_false", dest="make_report", default=True)
     parser.add_argument(
-        "--report-doc-id",
+        "--doc-id",
         help="Document number to write to the qualification test report",
         type=str,
-        default="E1200-0000-005",
+        default=DEFAULT_DOC_ID,
```

Comment:
I would rather leave out the default here, and then `if args.doc_id is None` determine a default.

Reply:
That's a fair point. Done.